ocaml/xenopsd/scripts/qemu-wrapper

```diff
@@ -248,20 +247,7 @@ def main(argv):
 
         if p == "-loadvm":
             loadvm_path = qemu_args[n+1]
-            if qemu_trad_image.is_trad_image(loadvm_path):
```

Comment:
I think it may be nice to check following which were done in `qemu_trad_image.is_trad_image` (before determination on if it is a traditional image):
```
    def load_header(self):
        self.magic = self.read_be32()
        if self.magic != QEMU_VM_FILE_MAGIC:
            self.error("not a qemu image")

        self.version = self.read_be32()
        if self.version != QEMU_VM_FILE_VERSION:
            self.error("save version %d not supported" % (self.version))
```
These checks are useful to current image as well.

Reply:
Now that we're no longer parsing the image in the wrapper, I think it would be better to just leave those checks to QEMU.